CHANGELOG.md

```diff
@@ -0,0 +1,154 @@
+- **transit** layer: Remove OSM transit features that have been "razed", "demolished", "removed", or "proposed". [#35](https://github.com/protomaps/basemaps/pull/35)
+- Fix project path in README.md [#24](https://github.com/protomaps/basemaps/pull/40)
+- Fix Markdown in link formatting [#24](https://github.com/protomaps/basemaps/pull/24)
+
+#### INTERNAL CHANGES
+
+- Update default Protomaps style in MapLibre JS so it shows off the new features (and any impactful changes). [#37](https://github.com/protomaps/basemaps/pull/37) and [#47](https://github.com/protomaps/basemaps/pull/47)
+- Update default Protomaps style in MapLibre JS so road casing layers are below other road layers [#42](https://github.com/protomaps/basemaps/pull/42)
+- Update default Protomaps style in MapLibre JS so country boundary lines are solid [#45](https://github.com/protomaps/basemaps/pull/45)
+- Update default Protomaps style in MapLibre JS with new "basic" color theme [#46](https://github.com/protomaps/basemaps/pull/46)
+- Update OpenLayers plugin example to use official olpmtiles library instead of custom library [#49](https://github.com/protomaps/basemaps/pull/49)
+- **landuse** layer: Allow fallback coallese of allow-listed OSM tags into `pmap:kind`. [#47](https://github.com/protomaps/basemaps/pull/47)
+- **roads** layer: Significant refactor of the kind and other property logic.
+- Add Usage section to README: [#20](https://github.com/protomaps/basemaps/pull/20)
+- Update Usage section to README for building tiles and applying code linting (formatting): [#37](https://github.com/protomaps/basemaps/pull/37)
+- Add Makefile with common build commands for easier development  [#47](https://github.com/protomaps/basemaps/pull/47)
+- Add ability to build GeoFabrik named `area` regions (eg "monaco") in the CLI. [#20](https://github.com/protomaps/basemaps/pull/20)
+- Improve consistency of internal private variable names in Planetiler profile   [#47](https://github.com/protomaps/basemaps/pull/47)
+- Add SonarCloud linting... [#31](https://github.com/protomaps/basemaps/pull/31)
+- Add Spotless [#25](https://github.com/protomaps/basemaps/pull/25)
+- Don't version track target, YARN, DStore, or PMTiles artifacts: [#20](https://github.com/protomaps/basemaps/pull/20)
+
+
+v2.1.0
+------
+- **Release date:** 2023-04-26.
+- **Credits:** [@bdon](https://github.com/bdon)
+
+- Initial open source release as a reimplementation of the Protomaps Basemap vector tile schema as a [Planetiler](https://github.com/onthegomap/planetiler/) schema in Java.
+- Add openlayers basemap example
+- Publish new PMTiles artifact:
+    - https://r2-public.protomaps.com/protomaps-sample-datasets/protomaps-basemap-opensource-20230408.pmtiles
+
+
+v2.0.0
+------
+- **Release date:** 2020-10-26
+- **Credits:** [@bdon](https://github.com/bdon)
+
+- Last closed source version of the Protomaps Basemap vector tile schema
+- Add build script
+
+
+v1.0.0
+------
+- **Release date:** 2021-01-01.
```

Comment:
@bdon Please update this with a real date

Reply:
updated